src/spec/SoundSpec.ts

```diff
@@ -1,457 +1,412 @@
 import * as ex from '@excalibur';
-import { Stubs } from './util/Stubs';
+import { WebAudio } from '../engine/Util/WebAudio';
 import { TestUtils } from './util/TestUtils';
 
 describe('Sound resource', () => {
   let sut: ex.Sound;
 
-  beforeAll(() => {
+  beforeAll(async () => {
     ex.Logger.getInstance().clearAppenders();
+    await WebAudio.unlock();
   });
 
   beforeEach(() => {
-    sut = new ex.Sound('/sample/test.mp3');
-
-    expect(sut.path).toBe('/sample/test.mp3');
+    sut = new ex.Sound('base/src/spec/images/SoundSpec/test.mp3');
```

Comment:
Is 'sut' an acronym or abbreviation? sound under test?

Reply:
It's an acronym for "subject under test", some people find it useful to identify what's actually being tested in a test method